### DISCUSSION TITLE
How to run this not on a ESP32-S3-BOX?

### DISCUSSION THREAD

**Discussion by Chrusciki** [toverainc/willow#99] [Q&A]
i cannot find a ESP32-S3-BOX instock anywhere, is there an option to run willow on anything else or locally on my PC?

I set up the willow interface server on my server, but unsure how to unwrap this project to run on my PC instead of the ESP32

**Comment by Chrusciki** [comment]
looking into https://github.com/toverainc/willow-ts-client

**Reply by kristiankielhofner** [reply]
Note that the willow-ts-client is for WebRTC interaction directly with the Willow Inference Server. It is meant for use in your own speech recognition web projects and does not contain wake word or any of the Willow elements that interact with Home Assistant, etc.

**Reply by Chrusciki** [reply]
Thanks for the heads up, so there is no other way to run willow with out a esp32-s3-box/lite?

**Reply by kristiankielhofner** [reply]
Correct. Willow specifically targets the ESP BOX series of hardware for the time being.

**Comment by grmrgecko** [comment]
By the way, I got an ESP Box fairly quickly via digikey when it wasn't in stock.
